core/tests/puppeteer-acceptance-tests/utilities/user/topic-manager.ts

```diff
@@ -1193,16 +1466,111 @@ export class TopicManager extends BaseUser {
     await this.expectElementToBeVisible(confirmMoveButton, false);
   }
 
+  /**
+   * Checks if the skill is visible in the skill selection modal.
+   * @param skillName The name of the skill.
+   * @param visible Whether the skill should be visible or not.
+   */
+  async expectSkillInSkillSelectionModalToBeVisible(
+    skillName: string,
+    visible: boolean = true
+  ): Promise<ElementHandle | null> {
+    const skillVisible = await this.isElementVisible(skillItem);
```

Comment:
start with a pre-check to make sure the page has loaded? Otherwise a "missing" skill might just not have loaded yet